python/cdk-validator-cfnguard/cdk_validator_cfnguard/cdk_validator_cfnguard_stack.py

```diff
@@ -0,0 +1,19 @@
+from aws_cdk import (
+    # Duration,
+    Stack,
+    # aws_sqs as sqs,
+)
+from constructs import Construct
+
+class CdkValidatorCfnguardStack(Stack):
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # The code that defines your stack goes here
+
+        # example resource
+        # queue = sqs.Queue(
```

Comment:
Did you mean to leave this commented out?

Reply:
Let me remove the boilerplate